pyproject.toml

```diff
@@ -72,11 +72,8 @@ numpy = "*"
 validobj = "*"
 prompt_toolkit = "*"
 frozendict = "*"  # validphys: needed for caching of data loading
-# Reportengine (and its dependencies) need to be installed in a bit more manual way
-# the version of RE that we need requires curio, and the pypi package for curio is no longer updated
-curio = { git = "https://github.com/dabeaz/curio", rev = "8667999" }
-reportengine = { git = "https://github.com/NNPDF/reportengine", rev = "3bb2b1d"}
-ruamel_yaml = {version = "<0.18"}
+# Reportengine needs to be installed from git
+reportengine = { git = "https://github.com/NNPDF/reportengine" }
```

Comment:
What's stopping us from releasing a package on pypi? Is Zahari the only person who can do this?

Reply:
Yes, it is under his account.

I can ask him to transfer it or free it or whatever once we have this working. But since nnpdf is also not in pypi this should be fine for now.